### PR TITLE
Update docs about session cookies and add config docs

### LINKED ISSUE #117: Update docs
**Upd:**

- Write docs about config
- <del>Update session docs (cookie)</del>

### PR DESCRIPTION
**Pull request by rnehra01**
Fix #117

### CODE REVIEW

**Comment by rnehra01** [comment]
@afeena In the docs I saw something like `:ref: session-manager`, looks strange to me. Is it intended?

**Comment by afeena** [comment]
@rnehra01 it's reference to the section -  [docs]( http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-anything)

I added some notes to the issue :)

**Comment by rnehra01** [comment]
@afeena Done

File: docs/source/config.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Tanner uses ``INI`` like format for configuration file. It's value can specified by using ``config`` flag
+
+There are 8 different sections :
+  * **DATA** 
+  
+    :db_config: Location of SQLI database configuration
+    :dorks: Location of dorks
+    :user_dorks: Location of user dorks
+    :vdocs: Location of configuration file for virtual docs
+  * **Tanner**
```
Comment (afeena) [inline docs/source/config.rst]: uppercase? :)

File: docs/source/config.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
+Configuration file
+=================
```
Comment (afeena) [inline docs/source/config.rst]: Warning :)
```
/home/afeena/tanner/docs/source/config.rst:2: WARNING: Title underline too short.

Configuration file
=================
```

**Comment by afeena** [comment]
Could you please include `config`  in `index.rst`?

**Comment by rnehra01** [comment]
@afeena Done.

**Pull request merged**